packages/client-app/src/components/ViewLayout/ViewLayout.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div class="flex xl:flex-row flex-col xl:gap-0 gap-1 custom-scroll rounded">
+  <div
+    class="m-1 mt-0 flex min-h-0 flex-1 rounded xl:rounded-lg xl:border xl:overflow-hidden leading-[19.5px]">
```

Comment:
😢

Reply:
i just moved what was already there, leading?

Reply:
i think its the pixel values 👀 maybe we can add a linter or smth

Reply:
yes I die a little inside every time

Reply:
yep there's a lint rule! Will add

https://www.npmjs.com/package/eslint-plugin-tailwindcss

Reply:
let me check that out real quick

Reply:
❤️

Reply:
updated. note to myself our variable are messing with default tw leading sizes, to be checked